POITableViewCell.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  POITableViewCell.swift
+//  GrailDiary
+//
+//  Created by Ben Gohlke on 10/10/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class POITableViewCell: UITableViewCell {
+    
+    
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var countryLabel: UILabel!
+    @IBOutlet weak var cluesCountLabel: UILabel!
+
+    
```

Comment:
We can get rid of this array here. We want just one source of data for our poi model